w4af/core/ui/api/utils/scans.py

```diff
@@ -27,6 +27,19 @@
 from w4af.core.ui.api.db.master import SCANS
 import w4af.core.controllers.output_manager as om
 
+import re
+
+
+def validate_profile_file(profile_name):
+    pattern = r'^\w+\.pw4af$'
+    return bool(re.match(pattern, profile_name))
+
+
+def validate_file_exist(profile_name):
+    cwd = os.getcwd()
+    file_string = f"{cwd}/profiles/{profile_name}"
```

Comment:
This would be safer / cleaner / more portable with `os.path.join()`. Any time you are using string formatting with user-supplied data to read from a file, I get nervous :)